models/users.js

```diff
@@ -306,6 +309,78 @@ const initializeUser = async (userId) => {
   return true;
 };
 
+/**
+ * Adds user data for verification by moderators
+ * @param userId {String} - RDS User Id
+ * @param discordId {Object} - Discord id of RDS user
+ * @param profileImageUrl {number} - profile image URL of user
+ * @param discordImageUrl {number} - discord image URL of user
+ */
+const addForVerification = async (userId, discordId, profileImageUrl, discordImageUrl) => {
+  try {
+    const isNotVerified = await photoVerificationModel.where("userId", "==", userId).limit(1).get();
+    const unverifiedUserData = {
+      userId,
+      discordId,
+      discord: { url: discordImageUrl, approved: false, date: admin.firestore.Timestamp.fromDate(new Date()) },
+      profile: { url: profileImageUrl, approved: false, date: admin.firestore.Timestamp.fromDate(new Date()) },
+    };
+    if (!isNotVerified.empty) {
+      const documentRef = isNotVerified.docs[0].ref;
+      // DOESN"T CHANGE THE APPROVAL STATE OF DISCORD IMAGE IF ALREADY VERIFIED
+      unverifiedUserData.discord.approved = isNotVerified.docs[0].data().discord.approved || false;
```

Comment:
visNotVerified.docs[0].data() is used at multiple places can be made into a constant

Reply:
Done